ApplicationServerDeployment/lambda/alexa/utils.py

```diff
@@ -0,0 +1,123 @@
+import base64
+import json
+import logging
+import requests
+import config
+logger = logging.getLogger(__name__)
+import os
+from response import AlexaResponse
+import jwt
+import time
+

```

Comment:
Avoid using utility functions where possible in favor of class methods. Add block comments to explain the purpose of the functions. Fix formatting adding new lines before all function definitions.

Reply:
refactored the code and moved files to a new class for the sidewalk app api